src/scss/styles.scss

```diff
@@ -0,0 +1,135 @@
+@import "../../node_modules/bootstrap/scss/bootstrap";
+@import "variables";
+
+
+
+// Login page
+
+#login-container {
+  margin-top: 10%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  .promt {
+    font-size: 18px;
+    font-weight: bold;
+    margin-bottom: 20px;
+    text-align: center;
+  }
+
+  #login-form{
+    display: flex;
+    justify-content: center;
+
+
+    input {
+      border: 3px solid;
+      border-radius: 4px;
+      transition: border-color 0.5s ease-out;
+  
+      &:invalid {
+        border-color: $form_error_color; // Make sure to define $form_error_color
+      }
+    }
+  }
+  }
+
+  
+
+  //profile page
+
+  .container-profile {
```

Comment:
These classes could be achieved with Bootstrap classes. Check bootstrap documentation for reference

Reply:
ajusted to use more bootstrap classes rather than using sass